ddtrace/context.py

```diff
@@ -51,6 +52,18 @@ def span_id(self):
         with self._lock:
             return self._parent_span_id
 
+    @property
+    def sampled(self):
+        """Return sampling status."""
```

Comment:
Actually we're providing this functionality with the `ctx.is_sampled()` method. Even if Properties are Python idiomatic, I would prefer to not creating this alias and just let people use the `is_sampled()`. Later, we can simply redesign part of the `Context` so that we can officially deprecate the method, and then remove it in a later change.

What do you think?

Reference: https://github.com/thehesiod/dd-trace-py/blob/4f30d5a178bfe30772811da0ffa243f2fd1092e9/ddtrace/context.py#L146-L151